R-proj/src/rotating.cpp

```diff
@@ -27,13 +27,15 @@
 //'
 //' @param P A convex polytope (H-, V-polytope or a zonotope).
 //' @param T Optional. A rotation matrix.
+//' @param seed Optional. A fixed seed for the random linear map generator.
 //'
 //' @section warning:
 //' Do not use this function.
 //'
 //' @return A matrix that describes the rotated polytope
 // [[Rcpp::export]]
-Rcpp::NumericMatrix rotating (Rcpp::Reference P, Rcpp::Nullable<Rcpp::NumericMatrix> T = R_NilValue){
+Rcpp::NumericMatrix rotating (Rcpp::Reference P, Rcpp::Nullable<Rcpp::NumericMatrix> T = R_NilValue,
```

Comment:
why not creating two overloads of `rotating`, one with fixed seed and one without ?

Reply:
We use the same code structure for polytope generators as well.